.github/workflows/CI2.yml

```diff
@@ -0,0 +1,90 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 3 * * *'
+
+jobs:
+  CI:
+    continue-on-error: ${{ matrix.kokkos_ver == 'develop' || matrix.distro == 'fedora:latest' || matrix.distro == 'fedora:intel' || matrix.distro == 'ubuntu:devel' }}
+    strategy:
+      matrix:
+        distro: ['fedora:latest', 'fedora:rawhide', 'opensuse:latest', 'ubuntu:latest', 'ubuntu:rolling'] # 'ubuntu:devel'
+        cxx: ['g++', 'clang++']
+        openmp: ['ON']
+        cmake_build_type: ['Debug', 'Release']
+        kokkos_ver: ['3.2.01', 'master', 'develop']
+        include:
+          - distro: 'fedora:intel'
+            cxx: 'icpc'
+            openmp: 'ON'
+            cmake_build_type: 'Release'
+            kokkos_ver: 'master'
+          - distro: 'fedora:intel'
+            cxx: 'icpc'
+            openmp: 'ON'
+            cmake_build_type: 'Debug'
+            kokkos_ver: 'master'
+          - distro: 'fedora:intel'
+            cxx: 'icpc'
+            cmake_build_type: 'Release'
+            openmp: 'OFF'
+            kokkos_ver: 'master'
+            cxx_flags: ' -parallel'
```

Comment:
Why is that needed? Especially because openmp is `OFF`.

Reply:
It's only for the `peakflops` tests. So something about vectorization

Reply:
I see, might be something to document.